packages/lens/src/alchemy.ts

```diff
@@ -0,0 +1,9 @@
+import { Alchemy, Network } from 'alchemy-sdk'
+
+const settings = {
+  apiKey: process.env.ALCHEMY_API_KEY,
+  network: Network.MATIC_MAINNET,
+  requestTimeout: 2000,
```

Comment:
I just put an arbitrary value as the requestTimeout. Thought was to let it timeout soonish and hit the reservoir backup.

Not sure if this is too much or too little.